lib/meetups_data.rb

```diff
@@ -47,4 +48,79 @@ def self.from_hash(data)
       self.new(data)
     end
   end
+
+  class GroupOrganizer
+    attr_reader :data
+
+    def initialize(data)
+      @data = data
+    end
+
+    def find_organizers
+
+      return if !data["url"].include? 'http://www.meetup.com/'
+
+      return if has_organizers?
+
+      group_urlname = parse_url_name_from_url(data["url"])
+      get_organizers(group_urlname)
+
+    end
+
+    def has_organizers?
+      data.has_key?("organizers")
+    end
+
+    def parse_url_name_from_url(meetup_url)
+      urlname = meetup_url.split('http://www.meetup.com/')
+      return urlname[1].to_s.chomp("/")
+    end
+
+    def get_organizers(meetup_urlname)
+      api_key = '7b6b15214f6e154232b7f3c6d417762'
+
+      conn = Faraday.new(:url => 'https://api.meetup.com') do |faraday|
+        faraday.request  :url_encoded
+        faraday.adapter  Faraday.default_adapter
+      end
+
+      group_response = conn.get '/2/groups', { :group_urlname => meetup_urlname, :key => api_key }
+
+      if (group_response.status != 200)
```

Comment:
This could also be broken out, and just take the response: `assert_ok(group_response)`